clap.zig

```diff
@@ -60,7 +60,8 @@ pub fn parseParam(line: []const u8) !Param(Help) {
     var res = Param(Help){
         .id = Help{
             .msg = line[0..0],
-            .value = line[0..0],
+            // see zig issue #4771
+            .value = &[0]u8{},
```

Comment:
The tests below fail otherwise when comparing the pointer value, see https://github.com/ziglang/zig/issues/4771